### PR TITLE
[Test] Order Encoding

### PR DESCRIPTION
**Pull request by bh2smith**
Ensure, via unit test, that contract interprets order data as expected.

With regards to PR #123

### CODE REVIEW

File: test/snapp_auction.js [hunk 1]
```diff
@@ -517,6 +517,24 @@ contract("SnappAuction", async (accounts) => {
 
       assert.equal(currentAuction.size, 2)
     })
+
+    it("Order Encoding", async () => {
```
Comment (fleupold) [inline test/snapp_auction.js]: nit: it's always nice when this reads as "it <does something>", Here I'd suggest "encodes order information in emitted event"

**Review by fleupold — APPROVED** [review]
Nice and easy. Feel free to merge directly into the other PR branch.

File: test/snapp_auction.js (hunk 1 above)
Comment (bh2smith) [reply test/snapp_auction.js]: Yes, that is cool I will make the change, but this doesn't mean the `it` in it refers to "it". I always assumed it meant "item".

**Review by josojo — APPROVED** [review]
Good test!

But I will have to refactor it immediately in my PR :). No worries, I can do it

File: test/snapp_auction.js (hunk 1 above)
Comment (bh2smith) [reply test/snapp_auction.js]:
```suggestion
    it("Encodes order information in emitted event", async () => {
```

**Comment by bh2smith** [comment]
@josojo Why do you need to change it?

**Pull request merged**